src/app/main/component/comments/services/comments.service.ts

```diff
@@ -6,7 +6,7 @@ import { AddedCommentDTO, CommentsModel } from '../models/comments-model';
   providedIn: 'root'
 })
 export abstract class CommentsService {
-  abstract addComment(entityId: number, text: string, parentCommentId: number): Observable<AddedCommentDTO>;
+  abstract addComment(entityId: number, text: string, imageFiles: File[], parentCommentId: number): Observable<AddedCommentDTO>;
```

Comment:
If a method accepts one parameter, it makes it easier to read, understand, and more flexible.
By using a wrapper of type FormData, you can encapsulate the parameters in a single object:
`abstract addComment(formData: FormData): Observable<AddedCommentDTO>;`
but note that it has 3 implementations